float64 base
float64 shoulder
float64 elbow